### ISSUE TITLE
Images generated are low quality

### ISSUE THREAD

**Issue by ivan-moto** [solothought/matchit#1]
Thanks for this app. It's great. 

The .png image generated by selecting "Download" are low quality. They're all less than 40 KB and grainy. Can you improve the quality?

Example:
![card_29](https://user-images.githubusercontent.com/13284085/71058738-adac4700-2115-11ea-912c-97166792d1d1.png)

**Comment by amitguptagwl** [comment]
@IvanMwiruki Agree. But I can't commit any date to fix it due to the issues in other opensource projects I'm maintaining. Please feel free to raise a PR. Otherwise, I'll look into sometime in future.

**Comment by ivan-moto** [comment]
Ok. No worries. Take your time. I found that if I select "Print" and save as a pdf I can get good quality images.
